test/12_nullablearray.jl

```diff
@@ -145,6 +145,45 @@ for (A, V, M) in ((NullableNominalArray, NullableNominalVector, NullableNominalM
             @test x[2] === eltype(x)()
             @test x[3] === eltype(x)()
             @test levels(x) == ["e", "c"]
+
+            push!(x, "e")
+            @test length(x) == 4
+            @test get(x[end]) == "e"
+            @test levels(x) == ["e", "c"]
+
+            push!(x, "zz")
+            @test length(x) == 5
+            @test get(x[end]) == "zz"
+            @test levels(x) == ["e", "c", "zz"]
+
+            push!(x, x[1])
+            @test length(x) == 6
+            @test get(x[1]) == get(x[end])
+            @test levels(x) == ["e", "c", "zz"]
+
+            push!(x, eltype(x)())
+            @test length(x) == 7
+            @test isnull(x[end])
+            @test levels(x) == ["e", "c", "zz"]
+
+            y = V{String, R}(levels(x))
+            append!(x, y)
+            @test length(x) == 10
```

Comment:
Here too, you can check the full contents of `x` using `isequal(x, NullableArray(["e", "c", ...]))`.